specification/columns/skumetername.md

```diff
@@ -0,0 +1,42 @@
+# SKU Meter Name
+
+The SKU Meter Name describes the capability being metered or measured by a particular SKU in a charge.
+
+Providers often have billing models in which multiple SKUs exist for a given service to describe and bill for different capabilities for that service. For example, an object storage service may have separate SKUs for capabilities such as object storage, API requests, data transfer, encryption, object management. This field helps practitioners understand which capabilities are being metered by the different SKUs that appear in a FOCUS dataset.
+
+The SkuMeterName column adheres to the following requirements:
+
+* SkuMeterName MUST be present in the billing data when when the provider includes a [SkuId](#skuid).
+* SkuMeterName MUST be of type String.
```

Comment:
Upfront, I realize that the answer here is probably, "we'll solve this in iteration", and this is not meant to block this pr.

I don't love how we're going to be combining coded strings (i.e. AWS) with clause-based strings (i.e. Azure, GCP).

Example
```
AWS: EU1-BoxUsage:m5.2xlarge
Azure: Native Metric Queries API Calls
GCP: N2D AMD Instance Core running in Frankfurt
```

_Ideally_, I'd push for a SkuMeterCode, SkuMeterName pairing where, with the examples above, AWS would populate `SkuMeterCode` and SHOULD provide a clause-based `SkuMeterName`, but Azure/GCP would populate `SkuMeterName` and SHOULD provide a code-based `SkuMeterCode`.

Working with data extensively across these 3 providers, I find classifying by AWS' usage types to be easily filterable because they usually follow a pattern, but I find them hard to decipher.  The opposite is true for Azure/GCP.  They are not easy to filter but easy to understand.

Reply:
would this not be better represented by the 'longer term solution' described in the supporting content?